packages/koa-liveness-ping/src/test/index.test.ts

```diff
@@ -0,0 +1,35 @@
+import {createMockContext} from '@shopify/jest-koa-mocks';
+import ping from '..';
+import mount from 'koa-mount';
+
+describe('koa-liveness-ping', () => {
+  it('calls next() when path is not `/ping`', async () => {
+    const middleware = ping();
+    const nextFn = jest.fn();
+    const ctx = createMockContext({url: '/some/other/path'});
+
+    await middleware(ctx, nextFn);
+
+    expect(nextFn).toHaveBeenCalledTimes(1);
+  });
+
+  it('returns with a 200 status when path is `/ping` and does not call next()', async () => {
+    const middleware = ping();
+    const nextFn = jest.fn();
+    const ctx = createMockContext({url: '/ping'});
+    await middleware(ctx, nextFn);
+
+    expect(ctx.status).toBe(200);
+    expect(nextFn).not.toHaveBeenCalled();
+  });
+
+  it('is mountable', async () => {
```

Comment:
Just thinking out loud if it makes sense to include this test. I like that we provide the suggestion in the Readme for using `koa-mount`, but I'm just wondering if we want to support it in this way. It is a third party package that we do not manage, additional dev dependency, etc... I am fine either way on it because koa packages are particularly stable and don't think there is any harm in this, I just wanted to mention it here.